src/app/api/services/initialization.py

```diff
@@ -0,0 +1,167 @@
+"""Author: Tadd Bindas"""
+
+from datetime import datetime
+from pathlib import Path
+from typing import Dict
+
+import geopandas as gpd
+import numpy as np
+import pandas as pd
+import yaml
+
+from app.core.settings import Settings
+
+
+def edit_yaml(original_file: Path, params: Dict[str, str], restart_file: Path) -> Path:
```

Comment:
This function is very much specific to the R&R t-route usage semantics, and also seems to miss an opporutnity to use the `troute.config` data models to handle the manipulation/updates, which would alleviate the need for this function to link a yaml dict with a params dict.